org.eclipse.xtext.ide/src/org/eclipse/xtext/ide/editor/quickfix/IQuickFixProvider.java

```diff
@@ -33,10 +36,22 @@ public interface IQuickFixProvider {
 	 *
 	 * @param options
 	 *            Contextual action options
-	 * @param diagnostic
-	 *            the diagnostic
+	 * @param fixMethods
+	 *            a list of methods that create the resolution
 	 * @return 0..n resolutions for the given issue
 	 */
 	List<DiagnosticResolution> getResolutions(Options options, Diagnostic diagnostic);
 
+	
+	/**
+	 *
+	 * Collects all of methods that create the resolution for the given diagnostic
```

Comment:
The JavaDoc doesn't match the implementation.

Is it possible to keep the original interface or make this a default method. Should be fine to always return true by default, shouldn't it?

Reply:
yes, it should be possible. done.